source/watchdog.py

```diff
@@ -70,6 +74,59 @@ def __getattr__(attrName: str) -> Any:
 _cancelCallEvent = None
 
 
+def _getCrashStatsPath() -> str:
```

Comment:
add to data class as a `@property`

Reply:
Done. Do you want me to roll the fingerprint annd timeout logic into a class too? The file here is pretty scattered with a lot done in functions so I didn't think it necessary to put these in classes but I can.